src/acl/group.test.ts

```diff
@@ -0,0 +1,1158 @@
+      "https://some.pod/group#webId",
+      "https://some.pod/container/",
+      { read: false, append: false, write: false, control: true },
+      "default"
+    );
+    const litDatasetWithJustResourceAcl = addAclDatasetToLitDataset(
+      litDataset,
+      resourceAcl,
+      "resource"
+    );
+    const litDatasetWithAcl = addAclDatasetToLitDataset(
+      litDatasetWithJustResourceAcl,
+      fallbackAcl,
+      "fallback"
+    );
+
+    const access = unstable_getGroupAccessOne(
+      litDatasetWithAcl,
+      "https://some.pod/group#webId"
+    );
+
+    expect(access).toEqual({
+      read: true,
+      append: false,
+      write: false,
+      control: false,
+    });
+  });
+
+  it("ignores default ACL rules from the Resource's own ACL LitDataset", () => {
+    const litDataset = getMockDataset("https://some.pod/container/");
+    const resourceAcl = addAclRuleQuads(
+      getMockDataset("https://some.pod/container/.acl"),
+      "https://some.pod/group#webId",
+      "https://some.pod/container/",
+      { read: true, append: false, write: false, control: false },
+      "resource"
+    );
+    const resourceAclWithDefaultRules = addAclRuleQuads(
+      resourceAcl,
+      "https://some.pod/group#webId",
+      "https://some.pod/container/",
+      { read: false, append: false, write: false, control: true },
+      "default"
+    );
+    const litDatasetWithAcl = addAclDatasetToLitDataset(
+      litDataset,
+      resourceAclWithDefaultRules,
+      "resource"
+    );
+
+    const access = unstable_getGroupAccessOne(
+      litDatasetWithAcl,
+      "https://some.pod/group#webId"
+    );
+
+    expect(access).toEqual({
+      read: true,
+      append: false,
+      write: false,
+      control: false,
+    });
+  });
+
+  it("ignores Resource ACL rules from the fallback ACL LitDataset", () => {
+    const litDataset = getMockDataset("https://some.pod/container/resource");
+    const fallbackAcl = addAclRuleQuads(
+      getMockDataset("https://some.pod/container/.acl"),
+      "https://some.pod/group#webId",
+      "https://some.pod/container/",
+      { read: true, append: false, write: false, control: false },
+      "resource"
+    );
+    const fallbackAclWithDefaultRules = addAclRuleQuads(
+      fallbackAcl,
+      "https://some.pod/group#webId",
+      "https://some.pod/container/",
+      { read: false, append: false, write: false, control: true },
+      "default"
+    );
+    const litDatasetWithAcl = addAclDatasetToLitDataset(
+      litDataset,
+      fallbackAclWithDefaultRules,
+      "fallback"
+    );
+
+    const access = unstable_getGroupAccessOne(
+      litDatasetWithAcl,
+      "https://some.pod/group#webId"
+    );
+
+    expect(access).toEqual({
+      read: false,
+      append: false,
+      write: false,
+      control: true,
+    });
+  });
+});
+
+describe("getAgentAccessAll", () => {
+  it("returns the Resource's own applicable ACL rules, grouped by Group WebID", () => {
```

Comment:
Same question about whether Groups have a WebID.

Reply:
Fixed in 2f3ede9b6e024a60563c7462b2db024d4f447345